blocks/search-results-list-block/themes/news.json

```diff
@@ -75,31 +78,34 @@
 				"components": {
 					"attribution": {
 						"display": "block",
-						"margin-top": "var(--global-spacing-4)"
+						"margin-block-start": "var(--global-spacing-4)"
 					},
 					"heading": {
 						"font-size": "var(--heading-level-5-font-size)",
 						"font-weight": "var(--heading-level-5-font-weight)",
-						"line-height": "var(--heading-level-5-line-height)",
-						"margin-bottom": "var(--global-spacing-4)"
+						"line-block-size": "var(--heading-level-5-line-height)",
```

Comment:
`line-height` is still the correct property to use. Could you revert all the places that was changed, thanks.

Reply:
@vgalatro reverted this [03f04dd](https://github.com/WPMedia/arc-themes-blocks/pull/1828/commits/03f04ddcc79cac1fad09c3b54fecc3c0856b705c)